src/chainparams.cpp

```diff
@@ -187,7 +187,17 @@ class CMainParams : public CChainParams
 
         // Note that of those with the service bits flag, most only support a subset of possible options
         vSeeds.clear();
-        vSeeds.emplace_back("seed.merit.me", false);
+        vSeeds.emplace_back("seed.merit.me");
```

Comment:
Need to set false for these at the moment as the second param.

Reply:
Oops.  Done